UnityProject/Assets/Scripts/Managers/CentComm.cs

```diff
@@ -158,6 +162,16 @@ IEnumerator WaitToGenericReport()
 			MakeAnnouncement(ChatTemplates.CentcomAnnounce, PlayerUtils.GetGenericReport(), UpdateSound.Announce);
 		}
 
+		private IEnumerator LowpopCheck()
+		{
+			yield return WaitFor.Seconds(Application.isEditor ? 60 : 300);
```

Comment:
doesn't this mean most of the crew wouldn't spawn without low pop access?

Reply:
Jack said he doesn't want everyone to spawn immediately with it so i added this timer before checking because people tend to late join 5 minutes usually so people dont sit on the lobby intentionally to cause everyone to get access.

Reply:
it doesn't ever update people. It asks GameManager if it is lowpop or not and uses either list of accesses accordingly. Now, reading the rest of the conversation, it sounds like you guys would rather have people keep their extra access even if the round isn't low pop anymore, is that correct?

Reply:
> it sounds like you guys would rather have people keep their extra access even if the round isn't low pop anymore, is that correct?

yes, taking it away mid round is a bad idea.